item.html

```diff
@@ -412,7 +382,11 @@ <h2 class="ftco-heading-2">Have a Questions?</h2>
 	};
   
 	  firebase.database().ref().update(updates);
-	  alert('Order Submitted!')
+	  alert('Order Submitted!');
+
+	  setTimeout(() => {
+		  window.location.href = 'services&product.html';
+	  }, 2000);
   }
   
   form.addEventListener('submit', writeOrder);
```

Comment:
This is some awesome practice for callback functions. Really impressed. keep up the good work. 👏